OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -48,6 +48,7 @@ public FrozenActor(Actor self, MPos[] footprint, CellRegion footprintRegion, Shr
 			isVisibleTest = shroud.IsVisibleTest(footprintRegion);
 
 			Footprint = footprint;
+			OccCells = Shroud.GetVisOrigins(actor).ToArray();
```

Comment:
Should I use "OccCells = self.OccupiesSpace.OccupiedCells().Select(c => c.First).ToArray();" instead of this line? Do I need the checks in GetVisOrigins()?